packer/linux/conf/bin/bk-install-elastic-stack.sh

```diff
@@ -62,13 +62,13 @@ export PLUGINS_ENABLED="${PLUGINS_ENABLED[*]-}"
 export BUILDKITE_ECR_POLICY=${BUILDKITE_ECR_POLICY:-none}
 EOF
 
-if [[ "${BUILDKITE_AGENT_RELEASE}" == "edge" ]] ; then
-	echo "Downloading buildkite-agent edge..."
-	curl -Lsf -o /usr/bin/buildkite-agent-edge \
-		"https://download.buildkite.com/agent/experimental/latest/buildkite-agent-linux-amd64"
-	chmod +x /usr/bin/buildkite-agent-edge
-	buildkite-agent-edge --version
-fi
+#if [[ "${BUILDKITE_AGENT_RELEASE}" == "edge" ]] ; then
+#	echo "Downloading buildkite-agent edge..."
+#	curl -Lsf -o /usr/bin/buildkite-agent-edge \
+#		"https://download.buildkite.com/agent/experimental/latest/buildkite-agent-linux-amd64"
+#	chmod +x /usr/bin/buildkite-agent-edge
+#	buildkite-agent-edge --version
+#fi
```

Comment:
why was this commented out? Shoudn't this be left here and match the architecture ?

Reply:
At the time I pushed the PR there was an issue with the prebuild packages not existing for aarch64.  This is fixed now so I'll rework this.